xmtp_mls/src/subscriptions/stream_all.rs

```diff
@@ -52,23 +52,36 @@ where
         conversation_type: Option<ConversationType>,
         consent_states: Option<Vec<ConsentState>>,
     ) -> Result<Self> {
-        let active_conversations = async {
+        let (active_conversations, sync_groups) = async {
             let provider = client.mls_provider()?;
             client.sync_welcomes(&provider).await?;
 
-            let active_conversations = provider
-                .conn_ref()
-                .find_groups(GroupQueryArgs {
-                    conversation_type,
-                    consent_states,
-                    include_duplicate_dms: true,
-                    ..Default::default()
-                })?
+            let groups = provider.conn_ref().find_groups(GroupQueryArgs {
+                conversation_type,
+                consent_states,
+                include_duplicate_dms: true,
+                include_sync_groups: conversation_type.is_none(),
```

Comment:
shouldn't it be `conversation_type.map(|c| c == ConversationType::Sync).unwrap_or(false)`?

If I set conversation type and consent states both to `None` I expect all messages, but also if i set conversation type to Sync I also expect just sync messages.

Reply:
Yes, but with `.unwrap_or(true)` I believe. Thanks for that catch. Updating.